registry/nextjs/components/select/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { useEffect, useRef, useState } from "react";
+import Icon from "../icon";
+import "@/registry/nextjs/components/select/select.css";
```

Comment:
relative imports are not allowed in the code base due to how the registry works. please use `@` base imports